openfecwebapp/templates/candidates-single.html

```diff
@@ -45,7 +45,7 @@ <h1 class="entity__name">{{ name }}</h1>
           </span>
           {% for c in committee_groups['P'] %}
           <span class="entity__term__data">
-            <a href="{{ url_for('committee_page', c_id=c.committee_id, cycle=c.related_cycle) }}">{{ c.name }}</a>
+            <a href="{{ url_for('committee_page', c_id=c.committee_id) }}">{{ c.name }}</a>
```

Comment:
Yep, exactly. Otherwise links could try to take you to a future cycle for a committee page that would return a 404.